website/static/scripts/tools.js

```diff
@@ -1,36 +1,58 @@
 (function() {
-  var $currentItem;
-  var $currentNav;
-  var $steps = $(".step");
-  var $stepHidden = $(".step-hidden");
+  let currentItem;
+  let currentNav;
+  const stepHidden = document.getElementsByClassName("step-hidden");
 
-  function reset() {
-    if ($currentItem) $currentItem.hide();
-    if ($currentNav) $currentNav.removeClass("active");
-    $currentItem = $currentNav = null;
-    $steps.removeAttr("style");
-    $stepHidden.removeAttr("style");
+  function hasClass(ele, cls) {
+    return ele.className.match(new RegExp("(\\s|^)" + cls + "(\\s|$)"));
   }
-
-  $(".tools-group .tools-button").click(function() {
-    if ($currentNav && $currentNav[0] === this) {
-      return reset();
-    } else {
-      reset();
+  function addClass(ele, cls) {
+    if (!hasClass(ele, cls)) {
+      ele.className += " " + cls;
     }
+  }
+  function removeClass(ele, cls) {
+    if (hasClass(ele, cls)) {
+      const reg = new RegExp("(\\s|^)" + cls + "(\\s|$)");
+      ele.className = ele.className.replace(reg, " ");
+    }
+  }
 
-    $currentNav = $(this);
-    $currentNav.addClass("active");
-
-    var name = $currentNav.attr("data-title");
-    location.hash = name;
-
-    $currentItem = $("[data-title=" + name + "]:not(.tools-button)").show();
-    $steps.show();
-    $stepHidden.show();
-  });
+  function reset() {
+    if (currentItem) {
+      for (let i = 0; i < currentItem.length; i++) {
+        currentItem[i].style.display = "none";
+      }
+    }
+    if (currentNav) {
+      removeClass(currentNav, "active");
+    }
+    currentItem = currentNav = null;
+    for (let i = 0; i < stepHidden.length; i++) {
+      stepHidden[i].removeAttribute("style");
+    }
+  }
 
-  if (location.hash && location.hash !== "#") {
-    $(".tools-button[data-title=" + location.hash.slice(1) + "]").click();
+  const buttons = document.querySelectorAll(".tools-group .tools-button");
```

Comment:
jQuery looks simpler for that part. Why do you want to remove jQuery on our website?

Reply:
Here's jQuery import by me. Extra request too much trouble.

Reply:
If you think it's ok to keep jquery since it's just our docs page that's fine too, but I suggested we could remove. From a code maintenance perspective I guess it is easier

Reply:
What do you think should be done here?